WordPress/build.gradle

```diff
@@ -124,6 +124,8 @@ android {
 
     flavorDimensions "app", "buildType"
 
+    def versionProperties = loadPropertiesFromFile(file("${rootDir}/version.properties"))
```

Comment:
All those changes in the build.gradle file are from https://github.com/wordpress-mobile/WordPress-Android/issues/15230 (which was merged in the release branch but we didn't merge release to develop since because we didn't do any new beta after that point)